Geometry_Engine/Compute/CollapseToPolylineEq.cs

```diff
@@ -79,7 +79,7 @@ private static Polyline Divide(this PolyCurve line, double shortestLength)
         private static Polyline Divide(this Arc arc, double shortestLength)
         {
             Polyline result = new Polyline();
-            double res = Math.Floor(arc.Length() / shortestLength);
+            double res = Math.Floor((shortestLength * 10 + 0.15 * arc.Length()) / shortestLength);
```

Comment:
Out of curiosity, where did these numbers come from? Are they part of the underlying theory?

Reply:
They are part of the underlying WIP and may not be questioned

Reply:
As long as they get documented properly by the time this is no longer WIP... :wink: